frontend/webpack.config.js

```diff
@@ -41,6 +42,9 @@ module.exports = (_env, argv) => ({
                     icon: true,
                 },
             }],
+        }, {
+            test: /\.css$/u,
+            use: ["css-loader"],
```

Comment:
```suggestion
            loader: "css-loader",
```

is a shortcut for this when there is only one loader and it doesn't need any options. We already use that, so we should be consistent here.